Vagrantfile

```diff
@@ -0,0 +1,25 @@
+require 'yaml'
+
+configuration = YAML.load(File.open(File.join(File.dirname(__FILE__), 'config.yaml'), File::RDONLY).read)
+
+Vagrant.configure(2) do |config|
+  source_path = configuration['sourcePath']
+  memory = configuration['memory']
+  web_port = configuration['webPort']
+  db_port = configuration['dbPort']
+
+  config.vm.box = 'ubuntu/trusty64'
```

Comment:
sorry, i misunderstood your question. so let's start this over again:

i'm using trusty because installing Docker via the Vagrant provider fails for Debian wheezy. I did not test this with jessie, maybe that works. let me just explore this option.